### PR TITLE
splitting CIS rhel7 profile - section 4

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- based on previous work, update the cis_rhel7.yml control file
- change rule references if needed

#### Rationale:

This is part of the effort which splits RHEL 7 CIS profile into 4 profiles based on levels.

### CODE REVIEW

File: controls/cis_rhel7.yml [hunk 1]
```diff
@@ -1226,3 +1226,322 @@ controls:
...
+
+  - id: 4.1.3
+    title: Ensure events that modify date and time information are collected (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - audit_rules_time_adjtimex
+    - audit_rules_time_clock_settime
```
Comment (yuumasato) [inline controls/cis_rhel7.yml:1310]: Rule `audit_rules_time_clock_settime` seems to have an extra filter `-F a0=0x0`.

File: controls/cis_rhel7.yml [hunk 2]
```diff
@@ -1226,3 +1226,322 @@ controls:
...
+
+  - id: 4.1.8
+    title: Ensure session initiation information is collected (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - audit_rules_session_events_wtmp
+    - audit_rules_session_events
```
Comment (yuumasato) [inline controls/cis_rhel7.yml:1364]: It seems to me that `audit_rules_session_events` covers `utmp`, `btmp` and `wtmp`.
And· there are also individual rules for each path, for example `audit_rules_session_events_utmp`

I suggest to go one approach or the other, but not overlap coverage.

**Review by yuumasato — APPROVED** [review]

**Comment by yuumasato** [comment]
Sorry for the delay in reacting to your changes.

**Comment by yuumasato** [comment]
@vojtapolasek gating failed on unique-cces test.

**Comment by vojtapolasek** [comment]
Thank you, fixed.

**Pull request merged**